packages/carousel/CHANGELOG.json

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
it doesn't seem right for this changelog to have all the changes from DS core

Reply:
My bad, I created the project by copying `core`.